pkg/server/container_start.go

```diff
@@ -283,7 +283,10 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 
 	// TODO(random-liu): [P1] Bind mount sandbox /dev/shm.
 
-	// TODO(random-liu): [P0] Bind mount sandbox resolv.conf.
+	// Bind mount sandbox resolv.conf.
+	if resolvPath := getResolvPath(getSandboxRootDir(c.rootDir, sandboxID), sandboxConfig); resolvPath != "" {
+		g.AddBindMount(resolvPath, resolvConfPath, []string{"ro"})
```

Comment:
Let's mount it as "rw" for now, and add TODO to figure out whether we should mount it as read-only.

Docker mount it as `ro`, and let's avoid regression for now, there may be user relying on this behavior. For example, there may be user doing hacky things with it.